daemon/src/test/java/org/mvndaemon/mvnd/builder/DependencyGraphTest.java

```diff
@@ -29,7 +29,7 @@ public void testRules() {
         MavenProject a = newProject("a"), b = newProject("b"), c = newProject("c");
```

Comment:
It may be better to rename the class altogether, given we're testing the SmartProjectDependencyGraph